storage/innobase/handler/ha_innopart.cc

```diff
@@ -1353,6 +1355,7 @@ ha_innopart::open(
 	if (m_ins_node_parts == NULL
 	    || m_upd_node_parts == NULL
 	    || m_blob_heap_parts == NULL
+		|| m_compress_heap_parts == NULL
```

Comment:
Formatting

Reply:
Fixed. This file is inconsistent in tabs and spaces usage.